control/tap-ctl-spawn.c

```diff
@@ -196,5 +232,15 @@ tap_ctl_spawn(void)
 	if (id < 0)
 		EPRINTF("get_id failed, child %d err %d\n", child, errno);
 
+	/* Put the tapdisk in a cgroup slice (best-effort) */
+	if (!slice) {
+#ifdef TAP_CTL_USE_DEFAULT_CGROUP_SLICE
```

Comment:
Yes, the intention was for it to be a compile-time switch. So if some people don't like the "move to the default slice" being the default behaviour they can easily disable it in one place. I felt this was more appropriate for now than adding another command-line option, especially as there are no users that will use that at present.